### PR TITLE
docs(portal-cli): update to match readme

### PR DESCRIPTION
**Pull request by HenriPro**
This doc is based on readme of repo, however changes made did not get reflected here.

### CODE REVIEW

**Comment by HenriPro** [comment]
I broke the link to https://github.com/Kong/kong-portal-templates =[

File: app/enterprise/1.3-x/developer-portal/helpers/cli.md [hunk 1]
```diff
@@ -31,32 +31,49 @@ For Kong Enterprise `<= 0.36`, or for `legacy mode` on Kong Enterprise `>= 1.3`
...
-* `wipe`     Deletes all content and themes from upstream workspace
+ Where `<workspace>` indicates the directory/workspace pairing you would like to operate on.
 
-Add `--watch` to make changes reactive
+### For `deploy`
+- Add `-W` or `--watch` to make changes reactive
+- Add `-P` or `--preserve` to avoid deleting files upstream that you do not have locally
 
+### For `fetch`
+- Add `-K` or `--keep-encode` to make keep binary assets as base64 encoded strings locally
```
Comment (GayleNeumann) [inline app/enterprise/1.3-x/developer-portal/helpers/cli.md]: Suggest removing "make" before "keep", or other way around, but not "make keep"
`- Add `-K` or `--keep-encode` to keep binary assets as base64 encoded strings locally`

File: app/enterprise/1.3-x/developer-portal/helpers/cli.md [hunk 2]
```diff
@@ -31,32 +31,49 @@ For Kong Enterprise `<= 0.36`, or for `legacy mode` on Kong Enterprise `>= 1.3`
...
 
 Make sure Kong is running and portal is on:
 
 Now from root folder of the templates repo you can run:
 
-```portal [-h,--help] [--config PATH] [-v,--verbose] <command>```
+```portal <command> <workspace>```
 
 Where `<command>` is one of:
+ - `config`    Output or change configuration of the portal on the given
```
Comment (GayleNeumann) [inline app/enterprise/1.3-x/developer-portal/helpers/cli.md]: on the given what?

File: app/enterprise/1.3-x/developer-portal/helpers/cli.md [hunk 3]
```diff
@@ -31,32 +31,49 @@ For Kong Enterprise `<= 0.36`, or for `legacy mode` on Kong Enterprise `>= 1.3`
...
+ - `deploy`    Deploy changes made locally under the given workspace upstream.
+ - `disable`   Enable the portal on the given workspace.
```
Comment (GayleNeumann) [inline app/enterprise/1.3-x/developer-portal/helpers/cli.md]: change to "Disable the portal on the given workspace."?

File: app/enterprise/1.3-x/developer-portal/helpers/cli.md [hunk 4]
```diff
@@ -31,32 +31,49 @@ For Kong Enterprise `<= 0.36`, or for `legacy mode` on Kong Enterprise `>= 1.3`
...
+ - `enable`    Enable the portal on the given workspace.
+ - `fetch`     Fetches content and themes from the given workspace.
+ - `serve`     Run the portal of a given workspace locally.
+ - `wipe`      Deletes all content and themes from upstream workspace
```
Comment (GayleNeumann) [inline app/enterprise/1.3-x/developer-portal/helpers/cli.md]: add a period after workspace.

File: app/enterprise/1.3-x/developer-portal/helpers/cli.md [hunk 5]
```diff
@@ -31,32 +31,49 @@ For Kong Enterprise `<= 0.36`, or for `legacy mode` on Kong Enterprise `>= 1.3`
...
-* `disable`  Enable the portal on the given workspace.
-* `enable`   Enable the portal on the given workspace.
-* `fetch`    Fetches content and themes from the given workspace.
-* `serve`    Run the portal of a given workspace locally.
-* `wipe`     Deletes all content and themes from upstream workspace
+ Where `<workspace>` indicates the directory/workspace pairing you would like to operate on.
 
-Add `--watch` to make changes reactive
+### For `deploy`
+- Add `-W` or `--watch` to make changes reactive
```
Comment (GayleNeumann) [inline app/enterprise/1.3-x/developer-portal/helpers/cli.md]: add period after reactive.

File: app/enterprise/1.3-x/developer-portal/helpers/cli.md [hunk 6]
```diff
@@ -31,32 +31,49 @@ For Kong Enterprise `<= 0.36`, or for `legacy mode` on Kong Enterprise `>= 1.3`
...
+- Add `-P` or `--preserve` to avoid deleting files upstream that you do not have locally
```
Comment (GayleNeumann) [inline app/enterprise/1.3-x/developer-portal/helpers/cli.md]: add period after locally.

File: app/enterprise/1.3-x/developer-portal/helpers/cli.md [hunk 7]
```diff
@@ -31,32 +31,49 @@ For Kong Enterprise `<= 0.36`, or for `legacy mode` on Kong Enterprise `>= 1.3`
...
+You can override config values set in `cli.conf.yaml` via environment variables.  If you wanted to override the kong admin url for example, you can run:
+
+```
+KONG_ADMIN_URL=http://new-admin-url.com portal deploy default
+```
+
+Environment variables are useful for scripting as well as temporarily overriding particular settings.
+
+Available environment variables include:
+  - `KONG_ADMIN_URL` Kong Admin URL the CLI should target for uploading files
```
Comment (GayleNeumann) [inline app/enterprise/1.3-x/developer-portal/helpers/cli.md]: add period after files.

File: app/enterprise/1.3-x/developer-portal/helpers/cli.md [hunk 8]
```diff
@@ -31,32 +31,49 @@ For Kong Enterprise `<= 0.36`, or for `legacy mode` on Kong Enterprise `>= 1.3`
...
+  - `KONG_ADMIN_TOKEN` Kong Admin Token token used to authenticate with the Kong Admin API
```
Comment (GayleNeumann) [inline app/enterprise/1.3-x/developer-portal/helpers/cli.md]: add period after API.

**Comment by HenriPro** [comment]
> @HenriPro I made some suggested changes. Please update as you see appropriate. Thx!

Awesome, wow I feel slightly illiterate 😂, good catches. I think I got all those now.

**Pull request merged**